openedx_events/event_bus/__init__.py

```diff
@@ -166,12 +168,11 @@ def make_single_consumer(*, topic: str, group_id: str, signal: OpenEdxPublicSign
     Arguments:
         topic: The event bus topic to consume from (without any environmental prefix)
         group_id: The consumer group to participate in
-        signal: Send consumed, decoded events to the receivers of this signal
+        signal: DEPRECATED signals determined by message headers
```

Comment:
If you add the notion of `signals determined by message headers` to a more permanent comment, then maybe the following would be more clear?
```suggestion
        signal: (DEPRECATED) this argument will be ignored.
```